charts/fluent-operator/templates/fluent-operator-deployment.yaml

```diff
@@ -95,7 +95,18 @@ spec:
         - name: env
           mountPath: /fluent-operator
       serviceAccountName: fluent-operator
+      {{- if .Values.operator.priorityClassName }}
+      priorityClassName: {{ .Values.operator.priorityClassName | quote }}
+      {{- end }}
       {{- if .Values.operator.imagePullSecrets }}
-      imagePullSecrets:
-      {{- toYaml .Values.operator.imagePullSecrets | nindent 8 }}
+      imagePullSecrets: {{ toYaml .Values.operator.imagePullSecrets | nindent 8 }}
+      {{- end }}
+      {{- if .Values.operator.tolerations }}
```

Comment:
maybe 
```
      imagePullSecrets: 
      {{ toYaml .Values.operator.imagePullSecrets | nindent 8 }}
      {{- end }}
```
better

Reply:
Indentation style is now aligned with the rest of the manifest.